keps/sig-node/2625-cpumanager-policies-thread-placement/README.md

```diff
@@ -0,0 +1,325 @@
+For these applications, just disabling SMT at machine level solves the need of the workload, but overall creates worse usage of hardware resources and poorer container density.
+
+Another policy option, or a further refinement of `smtalign`, which enables non-SMT emulation on SMT-enabled system would allow to accommodate these needs, but this would cause even more significant resource accounting mismatches
+as described above. Furthermore, at the moment of writing we are still assessing how large is the set of the classes which benefit of these extra guarantees.
+
+For all these reasons we postponed this work to a later date.
+
+### Test Plan
+
+The implementation PR will extend both the unit test suite and the E2E test suite to cover the policy changes described in this KEP.
+
+### Graduation Criteria
+
+#### Alpha
+- [X] Implement the new policy.
+- [X] Ensure proper e2e node tests are in place.
+
+#### Alpha to Beta Graduation
+- [X] Gather feedback from the consumer of the policy.
+- [X] No major bugs reported in the previous cycle.
+
+#### Beta to G.A Graduation
+- [X] Allowing time for feedback (1 year).
+- [X] Risks have been addressed.
+
+### Upgrade / Downgrade Strategy
+
+We expect no impact. The new policies are opt-in and separated by the existing ones.
+
+### Version Skew Strategy
+
+No changes needed
+
+## Production Readiness Review Questionnaire
+### Feature enablement and rollback
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [X] Feature gate (also fill in values in `kep.yaml`).
+    - Feature gate name: `CPUManagerPolicyOptions`.
+    - Components depending on the feature gate: kubelet
+  - [X] Change the kubelet configuration to set the cpumanager policy option to `smtalign`
+* **Does enabling the feature change any default behavior?**
+  - Yes, it makes the behaviour of the `cpumanager` static policy more restrictive and can lead to pod admission rejection.
+* **Can the feature be disabled once it has been enabled (i.e. can we rollback the enablement)?**
+  - Yes, disabling the feature gate shuts down the feature completely; alternatively,
+  - Yes, through kubelet configuration - switch to a different policy.
+* **What happens if we reenable the feature if it was previously rolled back?** No changes. Existing container will not see their allocation changed. New containers will.
+* **Are there any tests for feature enablement/disablement?**
+  - A specific e2e test will demonstrate that the default behaviour is preserved when the feature gate is disabled, or when the feature is not used (2 separate tests)
+
+### Rollout, Upgrade and Rollback Planning
+
+* **How can a rollout fail? Can it impact already running workloads?** Kubelet may fail to start. The kubelet may crash.
+* **What specific metrics should inform a rollback?** Not applicable
```

Comment:
What about pods failing to schedule / going into failed state?

Reply:
Right! will mention them there. Pod will be scheduled but it will end up in Failed/SMTAlignmenterror.